databox/api/src/Command/DocumentationDumperCommand.php

```diff
@@ -4,26 +4,104 @@
 
 namespace App\Command;
 
-use Alchemy\RenditionFactory\RenditionBuilderConfigurationDocumentation;
+use App\Documentation\DocumentationGeneratorInterface;
 use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\Attribute\AutowireIterator;
 
 #[AsCommand('app:documentation:dump')]
 class DocumentationDumperCommand extends Command
 {
+    /** @var array<string, DocumentationGeneratorInterface> */
+    private array $chapters = [];
+
     public function __construct(
-        private readonly RenditionBuilderConfigurationDocumentation $renditionBuilderConfigurationDocumentation,
+        #[AutowireIterator(DocumentationGeneratorInterface::TAG)] private readonly iterable $documentations,
     ) {
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        parent::configure();
+
+        /** @var DocumentationGeneratorInterface $documentation */
+        foreach ($this->documentations as $documentation) {
+            $k = $documentation->getPath();
+            if (isset($this->chapters[$k])) {
+                throw new \LogicException(sprintf('Chapter "%s" is already registered.', $k));
+            }
+            $this->chapters[$k] = $documentation;
+        }
+
+        $this
+            ->setDescription('Dump code-generated documentation(s)')
+            ->setHelp(sprintf('chapters: <info>%s</info>', join('</info> ; <info>', array_keys($this->chapters))))
```

Comment:
Not needed, `configure` method should not iterator over documentations, this is the execution phase.